pkg/provisioner/fss/fss.go

```diff
@@ -148,19 +152,20 @@ func (fsp *filesystemProvisioner) Provision(options controller.VolumeOptions, ad
 			},
```

Comment:
Above create a function-local logger `.With()` the `"availabilityDomain"` and `"fileSystemDisplayName"` fields above and log before calling `CreateFileSystem()`:

```go
logger.Info("Creating FileSystem")